packages/portfolio-contract/tools/plan-solve.ts

```diff
@@ -163,8 +163,8 @@ export const buildLPModel = (
     // The solution may be disrupted by either over- or under-weighting
     // variableFee w.r.t. fixedFee, but not otherwise, and we accept the risk.
     // TODO: Define RebalanceGraph['scale'] to eliminate this guesswork.
-    const magnifiedVariableFee = variableFee / 10_000;
-    const magnifiedFlatFee = fixedFee * 1e6;
+    const magnifiedVariableFee = variableFee / 10_000 + 1;
```

Comment:
Since expected values of `variableFee` are one or two digits, this will dramatically inflate the weighting of variable fees out of proportion with flat fees.
```suggestion
    const magnifiedVariableFee = variableFee / 10_000;
```
If something is broken with this, I'd like to get to the bottom of it.

Reply:
#12083